maps_dashboards/public/components/map_top_nav/get_top_nav_config.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { i18n } from '@osd/i18n';
+import { TopNavMenuData } from '../../../../../src/plugins/navigation/public';
+import {
+  OnSaveProps,
+  SavedObjectSaveModalOrigin,
+  showSaveModal,
+} from '../../../../../src/plugins/saved_objects/public';
+import { useOpenSearchDashboards } from '../../../../../src/plugins/opensearch_dashboards_react/public';
+import { MapServices } from '../../types';
+
+export const getTopNavConfig = () => {
+  const {
+    services: {
+      notifications: { toasts },
+      i18n: { Context: I18nContext },
+      savedObjects: { client: savedObjectsClient },
+    },
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
What rule are we breaking here?

Reply:
"getTopNavConfig" is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use"

Reply:
Ooooh right! Yes don't use the hook in this function. Just pass the services object as a prop to this function from the react component

Reply:
Yes, updated, thanks!